packages/material-ui/src/Badge/Badge.js

```diff
@@ -50,6 +51,14 @@ export const styles = theme => ({
     backgroundColor: theme.palette.error.main,
     color: theme.palette.error.contrastText,
   },
+  /* Styles applied to the badge `span` element if `hide={false}`. */
+  shown: {
```

Comment:
Most of the time, we only provide a class rule for the non-default state. I think that we can merge it with `root`.

Reply:
Okay :)